networkx/classes/tests/test_reportviews.py

```diff
@@ -873,7 +873,12 @@ def test_iterkeys(self):
         elist = sorted([(i, i + 1, 0) for i in range(8)] + [(1, 2, 3)])
         assert sorted(list(ev)) == elist
         # test order of arguments:graph, nbunch, data, keys, default
-        ev = evr((1, 2), "foo", True, 1)
+        ev = evr((1, 2), "foo", keys=True, default=1)
+        # test that the keyword arguments are passed correctly
```

Comment:
Let's clean up the comments here too. The leading comment line about `test of order arguments` doesn't even seem to be correct anymore. So maybe just move your new comment line up to replace the comment line on line 875.  That also puts line 876 under that description, which I think is accurate. Lots of flexibility here, so do what you think is best -- but I think the old line 875 should be removed/changed.

Reply:
Yes -- the `graph` is the part that was confusing before -- and the keys/default are no longer ordered. So probably best to just get rid of the claim to an ordered in a comment.

Reply:
Thanks, fixed!